packages/framework/fluid-static/src/types.ts

```diff
@@ -219,6 +245,8 @@ export interface IMember {
 }
 
 /**
- * An extended member object that includes currentConnection
+ * An extended member object that includes currentConnection.
+ *
+ * @public
  */
 export type Myself<M extends IMember = IMember> = M & { currentConnection: string };
```

Comment:
@zhenmichael While I'm here, I'm wondering if we can add a bit more context to this type. Looks like you maybe added this one? Can you offer some extra context on what `currentConnection` is?

Reply:
@Josmithr currentConnection is the container clientId associated with the member instance. Specifically, it is returned as this.container.clientId where container is defined in a ServiceAudience constructor.

Reply:
We don't appear to expose the `clientId` concept in the `IFluidContainer` API. Is there a more explicit way of expressing the semantics of `currentConnection` apart from the `IContainer` implementation details?